client/src/partials/inventory/configuration/units/units.html

```diff
@@ -0,0 +1,41 @@
+<div class="panel panel-default">
+  <div class="flex-heading">
+    <span class="flex-heading-label">
+      <i class="fa fa-list"></i>
+      {{ 'INVENTORY.LIST_UNIT' | translate }} ({{ UnitsCtrl.unitList.length }})
+    </span>
+
+    <button
+      ng-click="UnitsCtrl.addInventoryUnit()"
+      class="btn btn-primary btn-sm"
+      type="button"
+      data-create-unit><i class="fa fa-plus"></i>
+    </button>
+  </div>
+
+  <section style="height: 200px; overflow: auto;">
+    <table class="table table-condensed table-striped">
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>{{ 'FORM.LABELS.LABEL' | translate }}</th>
+          <th>&nbsp;</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr ng-repeat="unit in UnitsCtrl.unitList | orderBy: 'text'">
+          <td>{{ $index + 1 }}</td>
```

Comment:
Since these are units with ids, it would better to show the `id` instead of `$index`.  The `$index` can easily change (if someone else adds an inventory type, if the sort order changes, etc) so it's probably not a good idea to show the user.